apycula/tiled_fuzzer.py

```diff
@@ -89,6 +89,12 @@ def recode_idx_gw1n4(idx):
         "partnumber": "GW1N-LV9PG256C6/I5",
         "recode_idx": recode_idx_gw1n9,
     },
+    "GW1N-9C": {
+        "package": "UBGA332",
+        "device": "GW1N-9C-UBGA332-6",
+        "partnumber": "GW1N-LV9UG332C6/I5",
+        "recode_idx": recode_idx_gw1n9, # TODO: recheck
```

Comment:
Random sampling showed that the indices matched. For a total check we need a modified nextpnr.

Reply:
That's good news I suppose!